test/DotNetEnv.Tests/ParserTests.cs

```diff
@@ -340,6 +366,15 @@ public void ParseAssignment ()
                 Assert.Equal(value, kvp.Value);
             };
 
+            testParse("EV_DNE", "", "EV_DNE=#no value just comment");
+            testParse("EV_DNE", "", "EV_DNE= #no value just comment");
+            testParse("EV_DNE", "test", "EV_DNE= test #basic comment");
+            testParse("EV_DNE", "test", "EV_DNE= test  #a'bc allow singleQuotes in comment");
+            testParse("EV_DNE", "test", "EV_DNE= test  #a\"bc allow doubleQuotes in comment");
+            testParse("EV_DNE", "test", "EV_DNE= test  #a$bc allow dollarSign in comment");
```

Comment:
This one already passes, but seems like a good additional test

Reply:
Yes, needed some tests youbt these values if they still succeed with the new changes.